web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -215,12 +225,12 @@ def _get_office_hours_data(self):
         primary_industry_key = "mentor__expertprofile__primary_industry__name"
         office_hours = self._office_hours_queryset().filter(
             program__isnull=True).order_by(
-                 'start_date_time').annotate(
-                     finalist_count=Count("finalist")).annotate(
-                         reserved=Case(
-                             When(finalist_count__gt=0, then=Value(True)),
-                             default=Value(False),
-                             output_field=BooleanField()))
+            'start_date_time').annotate(
```

Comment:
why are we outdenting?

Reply:
this autoformatted my the IDE

Reply:
Okay. Can we fix it? Standard python style is to indent function args by one level (ie, four spaces) when they begin on the line following the function call.